emukit/test_functions/quadrature_functions.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+import numpy as np
+from scipy.integrate import quad, dblquad
+import matplotlib.pyplot as plt
```

Comment:
Let's remove plotting from here, matplotlib shouldn't be our core dependency

Reply:
see comment above. Then I'll also remove that dependency.